<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kargoTeslim</class>
 <widget class="QWidget" name="kargoTeslim">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>512</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>512</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Kargo Teslim</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/icon/icon.ico</normaloff>:/icon/icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
 	 background: #3498db;
	border-radius: 18px;
	font-family: Courier New;
	color: #ffffff;
	font-size: 15px;
	padding: 10px 20px 10px 20px;
	text-decoration: none;
    max-width:120px;
	min-width:120px;
	max-height:20px;	 
	min-height:20px;	 
}

QPushButton:hover {
	background: #3cb0fd;
	text-decoration: none;
}
QLineEdit{
	border: 2.5px double #55aaff;
    border-radius: 10px;
    background: rgb(255, 255, 255);
}
QComboBox{
	border: 2.5px double #55aaff;
    border-radius: 10px;
    background: rgb(255, 255, 255);
}
QWidget{
		font-family: Courier New;
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>515</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="lblMusteriKodu">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font:bold 18px
</string>
      </property>
      <property name="text">
       <string>Gönderici Müşterici Kodu :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lnMusteriKodu">
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>981</width>
     <height>298</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_13">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="lbl_AliciBilgileri">
        <property name="minimumSize">
         <size>
          <width>482</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>482</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font:bold 18px 
</string>
        </property>
        <property name="text">
         <string>Alıcı Bilgileri :</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>31</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lblAd">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>79</width>
              <height>31</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font:bold 18px 
</string>
            </property>
            <property name="text">
             <string>Ad :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lnAd">
            <property name="minimumSize">
             <size>
              <width>330</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>330</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius:10px;
font:16px </string>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Adınızı giriniz..</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lblSoyad">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>79</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font:bold 18px 
</string>
            </property>
            <property name="text">
             <string>Soyad :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lnSoyad">
            <property name="minimumSize">
             <size>
              <width>330</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>330</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius:10px;
font:16px </string>
            </property>
            <property name="placeholderText">
             <string>Soyadınızı giriniz..</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="lblCepNo">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>79</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font:bold 18px 
</string>
            </property>
            <property name="text">
             <string>Cep No :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lnCepNo">
            <property name="minimumSize">
             <size>
              <width>330</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>330</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius:10px;
font:16px </string>
            </property>
            <property name="inputMask">
             <string>(999)999 99 99</string>
            </property>
            <property name="text">
             <string>()  </string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="lblMail">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>79</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font:bold 18px 
</string>
            </property>
            <property name="text">
             <string>Mail :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lnMail">
            <property name="minimumSize">
             <size>
              <width>330</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>330</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius:10px;
font:16px</string>
            </property>
            <property name="placeholderText">
             <string>Mailinizi lütfen doğru giriniz..</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="lblAdres">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>79</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font:bold 18px 
</string>
            </property>
            <property name="text">
             <string>Adres :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lnAdres">
            <property name="minimumSize">
             <size>
              <width>330</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>330</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius:10px;
font:16px</string>
            </property>
            <property name="placeholderText">
             <string>Adresinizi giriniz..</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="lblAliciSube">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>79</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font:bold 18px
</string>
            </property>
            <property name="text">
             <string>Alıcı Şube :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cmbxSubeList">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius:10px;
font:16px </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="lblKargoBilgileri">
          <property name="minimumSize">
           <size>
            <width>482</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>482</width>
            <height>31</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font:bold 18px
</string>
          </property>
          <property name="text">
           <string>Kargo Bilgileri :</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>31</number>
          </property>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="lblAgirligi">
              <property name="minimumSize">
               <size>
                <width>110</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>79</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font:bold 18px 
</string>
              </property>
              <property name="text">
               <string>Ağırlığı :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lnAgirligi">
              <property name="minimumSize">
               <size>
                <width>220</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>440</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-radius:10px;
font:16px</string>
              </property>
              <property name="inputMask">
               <string/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="placeholderText">
               <string>Kargonun ağırlığını giriniz..</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="lblHacmi">
              <property name="minimumSize">
               <size>
                <width>110</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>79</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font:bold 18px
</string>
              </property>
              <property name="text">
               <string>Hacmi :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lnHacmi">
              <property name="minimumSize">
               <size>
                <width>220</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>440</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-radius:10px;
font:16px </string>
              </property>
              <property name="inputMask">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Kargonun hacmini giriniz..</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="lblUrunTipi">
              <property name="minimumSize">
               <size>
                <width>110</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>79</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font:bold 18px 
</string>
              </property>
              <property name="text">
               <string>Ürün Tipi :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lnUrunTipi">
              <property name="minimumSize">
               <size>
                <width>220</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>440</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-radius:10px;
font:16px</string>
              </property>
              <property name="placeholderText">
               <string>Ürünün tipini giriniz..</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="lblBirimFiyat">
              <property name="styleSheet">
               <string notr="true">font:bold 18px 
</string>
              </property>
              <property name="text">
               <string>Birim Fiyat :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lnBirimFiyat">
              <property name="styleSheet">
               <string notr="true">border-radius:10px;
font:16px </string>
              </property>
              <property name="text">
               <string>1.25</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QPushButton" name="btnKaydet">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Kaydet</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnGeri">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Geri</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lnAd</tabstop>
  <tabstop>lnSoyad</tabstop>
  <tabstop>lnCepNo</tabstop>
  <tabstop>lnMail</tabstop>
  <tabstop>lnAdres</tabstop>
  <tabstop>cmbxSubeList</tabstop>
  <tabstop>lnAgirligi</tabstop>
  <tabstop>lnHacmi</tabstop>
  <tabstop>lnUrunTipi</tabstop>
  <tabstop>lnBirimFiyat</tabstop>
  <tabstop>btnKaydet</tabstop>
  <tabstop>btnGeri</tabstop>
  <tabstop>lnMusteriKodu</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
